### DISCUSSION TITLE
how to build for a target architecture different than the host (e.g. using ARM to build for x64) ?

### DISCUSSION THREAD

**Discussion by Liz2Z** [coder/code-server#5223] [Q&A] [answered]
how to build for a target architecture different than the host (e.g. using ARM to build for x64) ?

i'm using M1 Mac.

**Comment by jsjoeio** [comment]
Hm...I'm not too sure. Let me see if @code-asher has any ideas on how to do this.

**Comment by Liz2Z** [comment, marked as answer]
I solved the problem with docker. Anyway, thanks.
